Week5/SOLUTION WEEK 5.py

```diff
@@ -0,0 +1,27 @@
+from csv import writer
+with open('student.csv','w', newline='') as a:
+    write=writer(a)
+    header=['sid','sname','city','contact']
+    write.writerow(header)
+
+    row=[[1,'Dhruv','Vyara',1234567888],
+              [2,'Harsh','Baroda',1234567890],
+              [3,'Trupal','Bardoli',1122334455],
+              [4,'Gopal','Amroli',2211334455],
+              [5,'Abhishek','Anand',3322114455]]
+    write.writerows(row)

```

Comment:
what is the use of writerows?

Reply:
It is used for writing multiple rows, we gave multiple values in row that we have to write in csv file. so writerows is used here while, to print single value, we can use writerow.